doc/man/8/rbd.rst

```diff
@@ -172,17 +171,17 @@ Commands
   require querying the OSDs for every potential object within the image.
 
 :command:`info` *image-spec* | *snap-spec*
-  Will dump information (such as size and order) about a specific rbd image.
+  Will dump information (such as size and object-size) about a specific rbd image.
```

Comment:
Probably can just say "object size" instead of "object-size" when not documenting the CLI arguments -- this applies to several spots below.